ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -304,34 +304,42 @@ def getFilterConfig(filterID):
   return gConfig.getOptionsDict('Resources/LogFilters/%s' % filterID)
 
 
-def getInfoAboutProviders(ofWhat=None, providerName=None, option='', section=''):
+def getInfoAboutProviders(of=None, providerName=None, option='', section=''):
```

Comment:
Maybe `providerType` rather than `of`?

Reply:
Definition of provider type already exist as OAuth, CAS, DRIACCA, Certificate and etc. Here means provider of what instance? Provider provide what? proxy, identity and etc.

Reply:
ah ok. Thanks for clarifying. I still think that `of` is not a good name though, especially in `if` conditions :D